### ISSUE TITLE
Explaination about what it checks.

### ISSUE THREAD

**Issue by ice-blaze** [Magentron/laravel-blade-lint#1]
I run `php artisan blade:lint` and get no errors. But in my code for example I have this kind of indentation:

```
<tr>
    <td>name1</td>
<td>name2</td>
</tr>
```

I can understand that indentation isn't the scope of laravel-blade-lint, but I'm curious to know what does it check?

**Comment by Magentron** [comment]
What it does is simply compiling blade templates to a temporary file and then running the PHP syntax checker (`php -l ...`) over it. So in theory if you have a PHP syntax error in your blade template, it should report on it. If not so, then please let me know.

**Comment by ice-blaze** [comment]
Thanks for the explanation :+1:
